autoscheduler/user_sessions/views.py

```diff
@@ -20,3 +23,19 @@ def set_last_term(request):
         return Response(status=400)
     request.session['term'] = term
     return Response()
+
+@api_view(['GET'])
+def get_full_name(request):
+    """ View that retrieves the first and last name seperated by a space for the user
+    of the current session """
+    user_id = request.session.get('_auth_user_id')
+    if user_id is None:
+        return Response(status=400)
+    user = User.objects.get(pk=user_id)
+    response = {'fullName': user.get_full_name()}
+    return Response(response)
+
+def logout_view(request):
+    """ Logs out the user and redirects to index"""
+    logout(request)
+    return redirect('/')
```

Comment:
At the top of the file if you import logout like this:
```python
from django.contrib import auth
```
we can avoid having to call this `logout_view`, since none of our other views have `view` in the name:
```suggestion
def logout(request):
    """ Logs out the user and redirects to index"""
    auth.logout(request)
    return redirect('/')
```